main/ertp/api/purse.md

```diff
@@ -1,78 +1,147 @@
+- Other kinds of rights, such as the right to participate in a particular contract.
+
+While each `purse` can only hold assets of one `brand`, any number of `purses` can be
+created that hold that `brand`. So you could have, say, three Quatloos `purses`, your
+friend Alice could have eight Quatloos `purses`, and so on. 
+
+You change a `purse`'s balance by calling either `deposit()` (to add assets)
+or `withdraw()` (to remove assets) on it. A `purse` can be empty, which if it holds 
+a fungible currency means it has a value of 0. If it holds non-fungible theatre tickets, 
+it means it just doesn't have any tickets.
+
+Unlike `payments`, `purses` are not meant to be sent to others. 
+To transfer digital assets, you should withdraw a `payment` from a `purse` and
+send the `payment` to another party.
+
+## issuer.makeEmptyPurse()
+- Returns: `{Purse}`
+
+While not a method called on a `purse`, clearly it's important to know how
+to create a new `purse`. Call `makeEmptyPurse()` on the `issuer` associated
+with the `brand` of assets you want the new `purse` to hold. It returns the
+new `purse` with an empty balance. 
+```js
+const {issuer: quatloosIssuer} = makeIssuerKit('quatloos');
+quatloosPurse1 = quatloosIssuer.makeEmptyPurse();
+quatloosPurse2 = quatloosIssuer.makeEmptyPurse();
+```
 
 ## purse.getCurrentAmount()
 - Returns: `{Amount}`
 
-Get the current `amount` contained in this purse, confirmed by the
-issuer. Note that this amount will change from call to call if assets
-have been deposited or withdrawn between calls. 
+Get an `amount` describing the current digital assets balance in the `purse`.
+This `amount` is confirmed by the `issuer` for the `brand` the `purse` can hold.
+Of course, the returned `amount` `value` might be different the next time you
+call `getCurrentAmount()` on the same `purse` if assets have been deposited or
+withdrawn from it in-between calls. . 
 
 ```js
-const { issuer } = makeIssuerKit('bucks');
-const purse = issuer.makeEmptyPurse();
-
-const currentBalance = purse.getCurrentAmount();
+const { issuer: quatloosIssuer } = makeIssuerKit('quatloos');
+const quatloosPurse = quatloosIssuer.makeEmptyPurse();
+// quatloos5 is a payment with balance of 5 quatloos
+const quatloosPurse.deposit(quatloos5);
+// Returns an amount with value = 5 and brand = quatloos
+const currentBalance = quatloosPurse.getCurrentAmount();
 ```
 
 ## purse.deposit(payment, optAmount)
 - `payment` `{Payment}`
-- `optAmount` `{Amount}` - Optional. This parameter ensures you are depositing the amount you expect.
+- `optAmount` `{Amount}` - Optional. 
 - Returns: `{Amount}`
 
-Deposit all the contents of `payment` into this purse, returning the payment's amount
-of digital assets (i.e. the deposit amount). If the optional argument `optAmount` does not equal the balance of
-`payment`, or if `payment` is an unresolved promise, throw an error.
+All I came up with that is specific to deposit was this:
```

Comment:
Is this an internal comment that accidentally got in??

Reply:
yep, that's my note to Tom. Shouldn't be here.